### PR TITLE
fix(controller): add validation for eval import/export

### PR DESCRIPTION
**Pull request by goldenxinxing**
## Description

1. add validation for import and export
2. tune request body

## Modules
- [ ] UI
- [x] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [x] run code format and lint check
- [x] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: server/controller/src/main/java/ai/starwhale/mlops/domain/ft/FineTuneAppService.java [hunk 1]
```diff
@@ -301,20 +306,45 @@ private FineTuneVo buildFineTuneVo(FineTuneEntity fineTuneEntity) {
...
+        // 1. status should be final
+        var jobs = jobMapper.findJobByUuids(uuids);
+        var validates = jobs.stream()
+                .filter(jobEntity -> jobStatusMachine.isFinal(jobEntity.getJobStatus()))
+                .map(JobEntity::getJobUuid)
+                .collect(Collectors.toList());
+        var success = evaluationRepo.migration(
                 String.format(TABLE_NAME_FORMAT, projectId),
-                uuids,
+                validates,
```
Comment (jialeicui) [inline server/controller/src/main/java/ai/starwhale/mlops/domain/ft/FineTuneAppService.java:319]: Server will response 4xx when `validates` is empty.

File: server/controller/src/main/java/ai/starwhale/mlops/domain/ft/FineTuneAppService.java [hunk 2]
```diff
@@ -301,20 +306,45 @@ private FineTuneVo buildFineTuneVo(FineTuneEntity fineTuneEntity) {
                 .build();
     }
 
-    public int importEvalFromCommon(Long projectId, Long spaceId, List<String> uuids) {
-        return evaluationRepo.migration(
+    public MigrationResult importEvalFromCommon(Long projectId, Long spaceId, List<String> uuids) {
+        // validate the src rows
+        // 1. status should be final
+        var jobs = jobMapper.findJobByUuids(uuids);
```
Comment (jialeicui) [inline server/controller/src/main/java/ai/starwhale/mlops/domain/ft/FineTuneAppService.java:312]: Find jobs with uuid and `projectId` will be better.

File: server/controller/src/main/java/ai/starwhale/mlops/domain/ft/FineTuneAppService.java (hunk 1 above)
Comment (goldenxinxing) [reply server/controller/src/main/java/ai/starwhale/mlops/domain/ft/FineTuneAppService.java:323]: done

File: server/controller/src/main/java/ai/starwhale/mlops/domain/ft/FineTuneAppService.java (hunk 2 above)
Comment (goldenxinxing) [reply server/controller/src/main/java/ai/starwhale/mlops/domain/ft/FineTuneAppService.java]: done

**Review by jialeicui — APPROVED** [review]
LGTM

**Pull request merged**